### PR TITLE
Templates/multimedia

### PR DESCRIPTION
**Pull request by digawp**
- [x] Add template for all multimedia details page, consists of preview, description, and review. Review may need to be reviewed (no pun intended)
- [x] ~~Add template for all multimedia list page~~

Decided not to implement multimedia list page first
Can merge if no problem.

Sample of usage is included for `book`

### CODE REVIEW

**Comment by digawp** [comment]
Decided not to implement multimedia list page first
Can merge if no problem

**Comment by digawp** [comment]
f wait let me fix this first, I made a huge mistake

**Comment by digawp** [comment]
ok can merge if no problem

File: multimedia/templates/multimedia/book_list.html [hunk 1]
```diff
@@ -1,6 +1,10 @@
 {% extends "multimedia/book_base.html" %}
 {% load staticfiles %}
 
+{% block title %}
+Books - Digital Content Store
```
Comment (9gix) [inline multimedia/templates/multimedia/book_list.html]: https://github.com/exclusive-gix/cs2102-store/blob/templates/multimedia/templates/index.html#L5

File: multimedia/templates/multimedia/book_detail.html [hunk 2]
```diff
@@ -1,38 +1,40 @@
-{% extends "multimedia/book_base.html" %}
+{% extends "multimedia/base_detail.html" %}
 {% load staticfiles %}
 
-{% block content %}
+{% block title %}
+{{ book.name }}
```
Comment (9gix) [inline multimedia/templates/multimedia/book_detail.html]: I guess this ain't gonna work on Movie, Music and Application.

**Comment by 9gix** [comment]
:+1:

**Pull request merged**